motoman_gp4_support/urdf/gp4_macro.xacro

```diff
@@ -141,13 +140,22 @@
       <axis xyz="-1 0 0" />
       <limit lower="${radians(-455)}" upper="${radians(455)}" effort="16.46" velocity="${radians(1000)}"/>
     </joint>
-    <joint name="${prefix}joint_6_t-tool0" type="fixed">
+	<link name="${prefix}flange"/>
+    <joint name="${prefix}joint_6_t-flange" type="fixed">
       <origin xyz="0.072 0 0" rpy="${radians(180)} ${radians(-90)} 0"/>
       <parent link="${prefix}link_6_t"/>
-      <child link="${prefix}tool0"/>
+      <child link="${prefix}flange"/>
     </joint>
     <!-- end of joint list -->
-
+	
+	<!-- default tool -->
```

Comment:
looks like these two lines have tabs instead of spaces

Reply:
Thanks for letting me know.  I just pushed a correction.